protocol/protos/NetRemoteWifi.proto

```diff
@@ -61,3 +61,39 @@ message WifiAccessPointDisableResult
     string AccessPointId = 1;
     WifiAccessPointOperationStatus Status = 2;
 }
+
+message WifiAccessPointSetPhyTypeRequest
+{
+    string AccessPointId = 1;
+    Microsoft.Net.Wifi.Dot11PhyType PhyType = 2;
+}
+
+message WifiAccessPointSetPhyTypeResult
+{
+    string AccessPointId = 1;
+    WifiAccessPointOperationStatus Status = 2;
```

Comment:
I think setting `WifiAccessPointOperationStatusCodeOperationNotSupported` is good enough, especially since we can set a status message alongside the status code. That should be enough information for the client to go off of in my opinion.

Reply:
Resolving.